src/schema/core.graphql

```diff
@@ -12,7 +12,7 @@ type Query {
   blocks(numbers: [Int], hashes: [String]): [Block]
 
   "Selects a range of blocks."
-  blocksRange(numberRange: [Int], hashRange: [String]): [Block]
+  blocksRange(numbers: [Int], hashes: [String]): [Block]
```

Comment:
Try not to change the schema. The schema is the contract with our clients, and we don't want to change it as the result of implementation details. Can we restore the previous field names?